odbc/src/api/functions.rs

```diff
@@ -1840,448 +1840,425 @@ pub unsafe extern "C" fn SQLGetInfoW(
+                        info_value_ptr as *mut WChar,
+                        buffer_length as usize,
+                        string_length_ptr,
+                    )
+                }
+                InfoType::SQL_CATALOG_TERM => {
+                    // MongoSQL uses the term "database".
+                    i16_len::set_output_wstring(
+                        "database",
+                        info_value_ptr as *mut WChar,
+                        buffer_length as usize,
+                        string_length_ptr,
+                    )
+                }
+                InfoType::SQL_CONVERT_FUNCTIONS => {
+                    // MongoSQL only supports the CAST type conversion function.
+                    i16_len::set_output_fixed_data(&SQL_FN_CVT_CAST, info_value_ptr, string_length_ptr)
+                }
+                InfoType::SQL_NUMERIC_FUNCTIONS => {
+                    // MongoSQL supports the following numeric functions.
+                    const NUMERIC_FUNCTIONS: u32 = SQL_FN_NUM_ABS
+                        | SQL_FN_NUM_CEILING
+                        | SQL_FN_NUM_COS
+                        | SQL_FN_NUM_FLOOR
+                        | SQL_FN_NUM_LOG
+                        | SQL_FN_NUM_MOD
+                        | SQL_FN_NUM_SIN
+                        | SQL_FN_NUM_SQRT
+                        | SQL_FN_NUM_TAN
+                        | SQL_FN_NUM_DEGREES
+                        | SQL_FN_NUM_POWER
+                        | SQL_FN_NUM_RADIANS
+                        | SQL_FN_NUM_ROUND;
+                    i16_len::set_output_fixed_data(&NUMERIC_FUNCTIONS, info_value_ptr, string_length_ptr)
+                }
+                InfoType::SQL_STRING_FUNCTIONS => {
+                    // MongoSQL supports the following string functions.
+                    const STRING_FUNCTIONS: u32 = SQL_FN_STR_CONCAT
+                        | SQL_FN_STR_LENGTH
+                        | SQL_FN_STR_SUBSTRING
+                        | SQL_FN_STR_BIT_LENGTH
+                        | SQL_FN_STR_CHAR_LENGTH
+                        | SQL_FN_STR_CHARACTER_LENGTH
+                        | SQL_FN_STR_OCTET_LENGTH
+                        | SQL_FN_STR_POSITION;
+                    i16_len::set_output_fixed_data(&STRING_FUNCTIONS, info_value_ptr, string_length_ptr)
+                }
+                InfoType::SQL_SYSTEM_FUNCTIONS => {
+                    // MongoSQL does not support any of the ODBC system functions.
+                    i16_len::set_output_fixed_data(&SQL_U32_ZERO, info_value_ptr, string_length_ptr)
+                }
+                InfoType::SQL_TIMEDATE_FUNCTIONS => {
+                    // MongoSQL supports the following timedate functions.
+                    const TIMEDATE_FUNCTIONS: u32 = SQL_FN_TD_CURRENT_TIMESTAMP
+                        | SQL_FN_TD_EXTRACT;
+                    i16_len::set_output_fixed_data(&TIMEDATE_FUNCTIONS, info_value_ptr, string_length_ptr)
+                }
```

Comment:
[nit] We don't need these comments anymore after the enum name change :D

Reply:
You are right, I'll remove them!

Reply:
Done